cvat-ui/src/assets/google-cloud.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="1em" height="1em" viewBox="0 0 256 206" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If you get this file on the Internet, could you please also attach a source and license?